packages/neo-one-cli/src/InteractiveCLI.ts

```diff
@@ -314,18 +315,18 @@ export class InteractiveCLI {
     );
 
     const subscription = start$.subscribe({
-      error: (error) => {
+      error: async (error) => {
```

Comment:
This can't be made async - the caller doesn't await it. Instead below you should do `reportError(...).catch(() => shutdown).then(() => shutdown)`

Reply:
![image](https://user-images.githubusercontent.com/39564353/51338512-e09a1680-1a3e-11e9-8561-32e83ca05a9f.png)

is after removing the Promise.resolve() as noted below.

Reply:
changing order of then & catch works

thanks @danwbyrne